MaxText/inference_mlperf/offline_mode.py

```diff
@@ -31,12 +31,23 @@
 
 import mlperf_loadgen as lg
 # pylint: disable=no-name-in-module
+
+import warnings
+
+warnings.simplefilter("ignore", category=FutureWarning)
+
+import inspect
+
+current_dir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parent_dir = os.path.dirname(current_dir)
+sys.path.insert(0, parent_dir)
+
```

Comment:
Is this to suppress the JAX warnings or something else?

Reply:
37 is to suppress jax future warnings. Others are to include python modules from parent dir